MADRaS/agents/rllib/train_rllib_multi_agent.py

```diff
@@ -5,6 +5,23 @@
 from ray.rllib.agents.ppo.ppo_policy import PPOTFPolicy
 from ray.tune.logger import pretty_print
 from ray.rllib.env.multi_agent_env import MultiAgentEnv
+from MADRaS.envs.gym_madras_v2 import MadrasEnv
+
+
+class MADRaSRLLib(MultiAgentEnv):
```

Comment:
Lets stick to our convention of Camel Case for class names and make `MADRaSRLLib` --> `MadrasRllib`